hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/repair/TestFSORepairTool.java

```diff
@@ -15,7 +15,7 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package org.apache.hadoop.fs.ozone;
+package org.apache.hadoop.ozone.repair;
```

Comment:
Should be `org.apache.hadoop.ozone.repair.om` to match `FSORepairTool` and avoid the need for `@VisibleForTesting`.